src/controller/SetUpCodePairer.h

```diff
@@ -48,6 +48,22 @@ namespace Controller {
 
 class DeviceCommissioner;
 
+class SetUpCodePairerParameters : public RendezvousParameters
+{
+public:
+    SetUpCodePairerParameters() = default;
```

Comment:
What does that actually do?  For example, what does it set `interfaceId` to, since that has no initializer?

I guess we just need this for `mCurrentParameters`?

Reply:
I have remove `SetUpCodePairerParameters() = default;` and make `SetUpCodePairerParameters mCurrentParameters` be an `Optional< SetUpCodePairerParameters> mCurrentParameters`